security-council-rehearsals/templates/r5-guardian-override/.env

```diff
@@ -0,0 +1,9 @@
+# Modify these values according to the Rehearsal instructions
+COUNCIL_SAFE=0xAB23dE0DbE0aedF356af3F815c8B47D88575D82d
```

Comment:
This safe is the same as used in previous rehearsals, the SC lead prefers to use it.